### PR TITLE
Add Interop 2025 proposal submission banner

### PR DESCRIPTION
**Pull request by DanielRyanSmith**
This change adds a banner across the front page and interop dashboard page of wpt.fyi to direct users to more information about proposal submission for Interop 2025. 

This is a simple update, and any design or wording choices can be changed as needed or directed.
Screenshot 2024-08-13 at 4 11 47 PM

Note that this change should be cleanly reverted once the proposal period has ended.

### CODE REVIEW

**Comment by DanielRyanSmith** [comment]
@nairnandu FYI

**Comment by DanielRyanSmith** [comment]
You can interact with the changes here: https://2024-08-13-interop-202-dot-wptdashboard-staging.uk.r.appspot.com/interop-2024

Note that the banner is on the results page (landing page) as well as the interop dashboard page.

**Comment by DanielRyanSmith** [comment]
View the staging deployment here: https://2024-08-25-mobile-resu-dot-wptdashboard-staging.uk.r.appspot.com/interop-2024?mobileView

It might be worth changing this in some way to be even more explicit that these are mobile results and experimental results.

**Comment by DanielRyanSmith** [comment]
@nairnandu as a reminder, this change still requires the explainer link for Interop 2025 submissions, and should be ready besides that. 🙂

**Comment by nairnandu** [comment]
> @nairnandu as a reminder, this change still requires the explainer link for Interop 2025 submissions, and should be ready besides that. 🙂

Thank you, Daniel! The banner can link to the [2025 readme](https://github.com/web-platform-tests/interop/blob/main/2025/README.md) once the [PR](https://github.com/web-platform-tests/interop/pull/689) is merged

File: webapp/views/wpt-app.js [hunk 1]
```diff
@@ -68,13 +68,39 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
...
+          text-decoration: none;
+        }
       </style>
 
       <app-location route="{{route}}" url-space-regex="^/(results)/"></app-location>
       <app-route route="{{route}}" pattern="/:page" data="{{routeData}}" tail="{{subroute}}"></app-route>
 
       <wpt-header path="[[encodedPath]]" query="[[query]]" user="[[user]]" is-triage-mode="[[isTriageMode]]"></wpt-header>
 
+      <a href="https://example.com" target="_blank">
```
Comment (jcscottiii) [inline webapp/views/wpt-app.js:96]: You may want to add a similar comment here to remove this anchor tag when interop 2025 proposals are closed.

**Review by jcscottiii — APPROVED** [review]
LGTM

**Pull request merged**